src/main/java/nl/workingtalent/wtacademy/reservation/ReadReservationDto.java

```diff
@@ -0,0 +1,75 @@
+package nl.workingtalent.wtacademy.reservation;
+
+import java.time.LocalDate;
+
+import nl.workingtalent.wtacademy.loan.Loan;
+
+public class ReadReservationDto {
+
+	private long id;
+	private ReservationRequest reservationRequest;
+	private LocalDate requestDate;
+	private long book;
+	private long user;
+	private Long loan;
+
+	public ReadReservationDto(Reservation reservation) {
+		id = reservation.getId();
+		reservationRequest = reservation.getReservationRequest();
+		requestDate = reservation.getRequestDate();
+		book = reservation.getBook().getId();
+		user = reservation.getUser().getId();
+		// loan moet null kunnen zijn
+		Loan reservationLoan = reservation.getLoan();
+		loan = (reservationLoan != null) ? reservationLoan.getId() : null;
+	}
```

Comment:
In plaats van een enum type voor reservation request, is het misschien makkelijker als de frontend gewoon een string krijgt.

Reply:
Fixed.